tasks/Divergent Association Task.md

```diff
@@ -0,0 +1,54 @@
+# Summary
+Measuring creativity
+Participants are asked to come up with 10 nouns that are as different as possible.
+
+# References
+Olson, Jay A., et al. “Naming Unrelated Words Predicts Creativity.” PNAS, National Academy of Sciences, 22 June 2021, www.pnas.org/content/118/25/e2022340118. 
+
+Link: https://www.pnas.org/content/118/25/e2022340118
+
+# Stimuli
+## The visual components
+N/A
+
+## Materials for alternative versions of the experiment 
+N/A
+
+# Procedure
+## Experiment 1
+The group of participants was selected entirely from undergraduate psychology courses from Melbourne, Australia, and from social media advertisements in Montreal, Canada.  The group was mostly female, between 18 and 20.
+
+This group follows the steps outlined.
+
+## Experiment 2
+The group of participants were entirely recruited over television, radio and social media advertisements as a part of a campaign by the Australian Broadcasting Corporation.  This was a much larger group, of 8,572 participants from 98 different countries.  Most were Australian or from the United Kingdom.  The reported ages ranged from 7 to over 70, most falling in-between 35 and 54.  The group was a majority female.
+
+This group follows the steps outlined.
+## Steps
+1. ask a participant to give 10 nouns that are as different from each other as possible
+2. take the first seven valid words given by the participant
+3. DAT score is calculated using the instructions in the Criteria section
+
+## Roles 
+Participant, gives 10 nouns that are as different as possible.
+
+## Instructions
+Participants are instructed to give 10 nouns that are as different as possible.
+
+The difference of the words is defined by the meanings of the words.
```

Comment:
Instructions
Please write 10 words that are as different from each other as possible, in all meanings and uses
of the words.
Rules
1. Only single words.
2. Only nouns (e.g., things, objects, concepts).
3. No proper nouns (e.g., no specific people or places).
4. No specialised vocabulary (e.g., no technical terms).
5. Think of the words on your own (e.g., do not just look at objects in your surroundings).
6. You will have 4 min

** Actual Instruction document included in stimuli section